lib/hammer_cli/output/adapter/base.rb

```diff
@@ -13,6 +13,8 @@ def print_record(fields, record)
     end
 
     def print_collection(fields, collection)
+      fields = field_filter.filter(fields)
+      fields = displayable_fields(fields, collection.first)
```

Comment:
It was missing on purpose. This functionality is disabled for table and csv adapters because we can't tell what data are in the other records. Using the first record for hiding fields that are missing would be probably fair guess, but the same method can remove also fields that are blank (`""`). That could result in some false positives -> whole column would be hidden just because the first record contained empty string.